src/systems/isq/include/units/isq/mechanics.h

```diff
@@ -38,7 +37,7 @@ QUANTITY_SPEC(surface_mass_density, mass / area);
 inline constexpr auto surface_density = surface_mass_density;
 QUANTITY_SPEC(linear_mass_density, mass / length);
 inline constexpr auto linear_density = linear_mass_density;
-QUANTITY_SPEC(momentum, mass* velocity);
+QUANTITY_SPEC(momentum, mass* velocity);      // TODO vector?
```

Comment:
`// TODO vector?`

It's defined as a vector. Isn't that implied by the expression? I thought it was calculated as such when not specified.

Reply:
Actually, it is not defined explicitly as a vector in ISO 80000, and typically, this document says explicitly "vector (ISO 80000-2) quantity ...". In this case, it does not. It only mentions it is a product of velocity, which is defined as a vector. The comment is to confirm it should not be explicitly overridden to a scalar. Although, with my poor knowledge of physics, I think momentum is a vector quantity indeed ;-)

And yes, the library calculated it correctly.